extensions-core/stats/src/test/java/org/apache/druid/query/aggregation/variance/sql/VarianceSqlAggregatorTest.java

```diff
@@ -724,4 +729,20 @@ public void testOverWindow()
         ))
         .run();
   }
+
+  @Test
+  public void testStddevNotSupportedOverWindow()
+  {
+    assumeFeatureAvailable(EngineFeature.WINDOW_FUNCTIONS);
+
+    DruidException e = assertThrows(
+        DruidException.class,
+        () -> testBuilder()
+            .queryContext(ImmutableMap.of(PlannerContext.CTX_ENABLE_WINDOW_FNS, true))
+            .sql("SELECT stddev(m1) OVER () from numfoo")
+            .run()
+    );
+
+    assertTrue(e.getMessage().contains("Aggregation [STDDEV] is currently not supported for window functions"));
```

Comment:
`assertTrue` in this form doesn't give much to go on...
either pass the full `e.getMessage()` as the error description; use `assertThat` like in the other tests